src/wifiman_msg.h

```diff
@@ -0,0 +1,68 @@
+/**
+ * @file wifiman_msg.h
+ * @author TheSomeMan
+ * @date 2020-11-07
+ * @copyright Ruuvi Innovations Ltd, license BSD-3-Clause.
+ */
```

Comment:
A brief description about usage of functions here would be good. Does the application need to know about this file, or is it used only internally?

If application needs to call these functions, how, when and why?

Reply:
> I noticed that at least some GW firmware versions (1.2.0?) do not refresh list of access points. The browser requests a list of hotspots to connect to, but the list is always the same regardless of e.g. mobile hotspot being turned on or off.
> 
> Does this pull request fix that? If not I can open a separate issue

No, this pull request fixes only bugs found by sonarcloud

Reply:
> A brief description about usage of functions here would be good. Does the application need to know about this file, or is it used only internally?
> 
> If application needs to call these functions, how, when and why?

this file is an internal header, 
in next pull request I'm going to add "include" folder and move all public APIs there, so I think it's not important to add any additional description here.

Reply:
Ok, Thanks. I'll mark this one as approved then.